### PR TITLE
Support `calypso/config` in webpack-config-flag-plugin

### PR DESCRIPTION
**Pull request by scinos**
### Background

We have a custom plugin `webpack-config-flag-plugin` that looks for occurences of `config.isEnabled(...)` and replaces it with a predefined result. 

This plugin works by looking for imports of the module `'config'`. As such, it doesn't work when importing from `calypso/config`.

At the moment we are only using it to replace `config.isEnabled("isDesktop")` (https://github.com/Automattic/wp-calypso/blob/master/client/webpack.config.js#L327)

### Changes proposed in this Pull Request

* Change the plugin to work on imports `config` or `calypso/config`
* Change the test fixtures for testing with `calypso/config` as well

### Testing instructions

Verify test still passes

### CODE REVIEW

File: packages/webpack-config-flag-plugin/.eslintrc.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ module.exports = {
 			files: [ '**/fixtures/**/*' ],
 			rules: {
 				'wpcalypso/no-relative-imports': 'off',
+				'import/no-extraneous-dependencies': 'off',
```
Comment (scinos) [inline packages/webpack-config-flag-plugin/.eslintrc.js:11]: Do not complain when a fixture imports from `calypso/config`

File: packages/webpack-config-flag-plugin/test/index.js [hunk 2]
```diff
@@ -19,37 +19,41 @@ describe( 'webpack-config-flag-plugin', () => {
...
+					minimize: false,
+				},
+				output: {
+					path: buildDirectory,
+					pathinfo: false,
+				},
+				resolve: {
+					extensions: [ '.js' ],
+					modules: [ fixturesDirectory ],
+					alias: {
```
Comment (scinos) [inline packages/webpack-config-flag-plugin/test/index.js:42]: Use an alias so we don't have to worry about `calypso/config` not existing.

File: packages/webpack-config-flag-plugin/index.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ class ConfigFlagPlugin {
 			parser.hooks.import.tap( 'ConfigFlagPlugin', ( statement, source ) => {
 				const currentModule = parser.state.current.resource;
 
-				if ( source === 'config' ) {
+				if ( source === 'config' || source === 'calypso/config' ) {
```
Comment (scinos) [inline packages/webpack-config-flag-plugin/index.js:47]: This is the core of the change. Everything else is adding or cleaning up tests.

**Review by sgomes — APPROVED** [review]
LGTM, great work @scinos! 👍 

I wish I had found a way to make that check less fragile, so that PRs like this one wouldn't have been necessary in the first place 😞

**Comment by scinos** [comment]
> I wish I had found a way to make that check less fragile

I can't figure out a better way either :(

**Pull request merged**